x-pack/plugin/src/test/resources/rest-api-spec/test/ml/index_layout.yml

```diff
@@ -1,742 +0,0 @@
-setup:
```

Comment:
All these tests are now covered in: `org.elasticsearch.xpack.ml.integration.IndexLayoutIT`